BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/location/distance/DistanceUtilTest.java

```diff
@@ -23,14 +23,17 @@ public void speedFilter_locations_accurateLocation() throws Exception {
         Location actualLocation = DistanceUtil.speedFilter(oldLocation, newLocation, 2);
         assertEquals(expectedLocation.getLatitude(), actualLocation.getLatitude(), 0);
         assertEquals(expectedLocation.getLongitude(), actualLocation.getLongitude(), 0);
+    }
 
+    @Test
+    public void speedFilter_smallDistance_accurateLocation() throws Exception {
         // smaller distance than speed filter
-        oldLocation = LocationPredictorTest.GENDARMENMARKT;
+        Location oldLocation = LocationPredictorTest.GENDARMENMARKT;
         oldLocation.setTimestamp(0);
-        newLocation = oldLocation.getShiftedLocation(1, 0);
+        Location newLocation = oldLocation.getShiftedLocation(1, 0);
         newLocation.setTimestamp(1000);
-        expectedLocation = oldLocation.getShiftedLocation(1, 0);
-        actualLocation = DistanceUtil.speedFilter(oldLocation, newLocation, 2);
+        Location expectedLocation = oldLocation.getShiftedLocation(1, 0);
+        Location actualLocation = DistanceUtil.speedFilter(oldLocation, newLocation, 2);
```

Comment:
How is this test different compared to `speedFilter_largeDistance_accurateLocation`?

When writing tests, you want to cover as many (edge-) cases as possible. E.g.:
- Speed from location A to B is higher than maximum speed > Correct change
- Speed from location A to B is exactly maximum speed > No change
- Speed from location A to B is lower than maximum speed > No change
- Location A equals B > No change